### ISSUE TITLE
Problem with BNN without BN

### ISSUE THREAD

**Issue by NguyenSakura** [NeuroCompLab-psu/SNN-Conversion#3]
Dear Author, 
 I followed your idea in this paper: Exploring the connection between binary and spiking neural networks. As mention in the paper. The training BNN without Batchnorm gives the performance of B-SNN is better. However, I try to run the BNN code (referred by you in Github for XNOR-Net) without Batchnorm. Unfortunately, the accuracy ( for example for Cifar10 with VGG) always below 10 %. I confused about BNN without Batchnorm. If you don't mind can you shared your BNN train and BNN conversion to B-SNN using VGG-16 for Cifar 100 as mentioned in your paper. I greatly appreciate it.  Many Thanks
